woocommerce/woocommerce
# [e2e] Expand simple product creation with more details

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce-quality/issues/606

This is an update to the existing `can create simple non-virtual product` test. We're expanding creating of a simple non-virtual product with:
- description
- short description
- SKU
- dimensions (weight, height, length, width)
- tag
- category

Additionally, in this commit [0ae5cd2](https://github.com/woocommerce/woocommerce/pull/43710/commits/0ae5cd219bc6f4fb2f1a17ab835b894d30ea2bfa) I added to API calls in the `afterAll` function to delete category and tag we added in this test. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

To be sure that the test is passing, run the existing spec: `pnpm test:e2e-pw ./tests/e2e-pw/tests/merchant/create-simple-product.spec.js`

CI jobs should pass as well.

To test category and tag removal, try the following:
- Run the test and pause it at the end (for example [here](https://github.com/woocommerce/woocommerce/pull/43710/files#diff-4d8b4fa2d3fc09e284b17868562b01cf7bc2d5294477590b31e45f0f01277ebbR193))
- Verify that category `nonVirtualCategory` and tag `nonVirtualTag` exists for the user
- Run the test again, but this time pause it before category and tag are added
-  Verify that category `nonVirtualCategory` and tag `nonVirtualTag` **don't** exist anymore



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
